src/components/Modals/MapModal.vue

```diff
@@ -19,7 +19,7 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
             <button @click="$refs.importInput.click()" id="import">Open Floorplan</button>
             <button @click="mapEnabled = false; mapVisible = false; $emit('close')">New Floorplan</button>
-            <button @click="mapEnabled = true; tool='Map'; $emit('close')">New Floorplan w/ Map</button>
+            <button @click="mapEnabled = true; tool='Map'; $emit('close')" :disabled="!online">New Floorplan w/ Map</button>
```

Comment:
Can we add a button[disabled] style? It doesn't look visually different, and still has `cursor: pointer`